tests/app/service/test_rest.py

```diff
@@ -365,6 +367,15 @@ def test_create_service_free_sms_fragment_limit_is_optional(client, sample_user)
         headers=headers)
     json_resp = json.loads(resp.get_data(as_text=True))
     assert resp.status_code == 201
+
+    # Test data from the new annual billing table
+    service_id = json_resp['data']['id']
+    annual_billing = client.get('service/{}/billing/free-sms-fragment-limit?financial_year_start={}'
+                                .format(service_id, get_current_financial_year_start_year()),
+                                headers=[('Content-Type', 'application/json'), create_authorization_header()])
+    json_resp = json.loads(annual_billing.get_data(as_text=True))
+    assert json_resp['data']['free_sms_fragment_limit'] == 9999
+    # TODO: Remove this after the new data is used
```

Comment:
Not sure on the TODOs. What is our policy? I prefer a new story in the backlog as you know if needs to be done? but thats me.

Reply:
Should be done at the end of the story to remove the existing mechanism. So won't be left for too long

Reply:
we don't really have a policy on todos - they don't have parity with pivotal issues. maybe they should, but sometimes they're just "I accept that this code is not perfect" tech debt flags, as opposed to things we're committed to revisiting.